### PR TITLE
New package: libyang-0.16r3

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/libyang/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'libyang'
+pkgname=libyang
+version=0.16r3
+revision=1
+_ver="0.16-r3"
```
Comment (Chocimier) [inline srcpkgs/libyang/template]: Better to have version defined once, please change like here: https://github.com/void-linux/void-packages/blob/561b3db2509f0ec6807819361d4c5fbe5230d1cb/srcpkgs/android-tools/template#L8

File: srcpkgs/libyang/template [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+wrksrc="${pkgname}-${_ver}"
```
Comment (Chocimier) [inline srcpkgs/libyang/template]: Please use `libyang` instead of `${pkgname}`.

File: srcpkgs/libyang/template [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+distfiles="https://github.com/CESNET/libyang/archive/v${_ver}.tar.gz"
+checksum=4745460dedc4ba17d8bcfc39ad9ba0d1b91bbe82b55b9417a090390909ba8ca5
+
+post_install() {
+	vlicense LICENSE
+}
+
+libyang-devel_package() {
+	short_desc+=" - development files"
+	depends="${srcpkgname}-${version}_${revision}"
```
Comment (Chocimier) [inline srcpkgs/libyang/template]: I have seen sourcepkg usually. Is srcpkgname something different or typo?

Comment (CameronNemo) [reply srcpkgs/libyang/template]: probably a typo

**Pull request merged**